src/engine/bank01.asm

```diff
@@ -7172,7 +7172,45 @@ Func_6cfa: ; 6cfa (1:6cfa)
 	ret
 ; 0x6d3f
 
-	INCROM $6d3f, $6d84
+Func_6d3f: ; 6d3f (1:6d3f)
+	or a
+	bit POISONED_F , [hl]
+	ret z
+	push hl
+	bit DOUBLE_POISONED_F, [hl]
+	ld a, PSN_DAMAGE
+	ldtx hl, Received10DamageDueToPoisonText    ; copies from the byte of the damage due to poison by 10
+	jr z, .Damage_Dealt_From_Poison
+	ld a, $14
```

Comment:
This `$14` still needs a constant (`DBLPSN_DAMAGE`)

Reply:
Fixed